config/sidekiq_scheduler.yml

```diff
@@ -223,3 +223,8 @@ CovidVaccine::ScheduledBatchJob:
   cron: "*/15 * * * * America/New_York"
   class: CovidVaccine::ScheduledBatchJob
   description: Batches received vaccine records and schedules upload to enrollment service.
+
+  CovidVaccine::ExpandedScheduledSubmissionJob:
+  cron: "*/15 * * * * America/New_York"
```

Comment:
Maybe schedule this to run 7.5. minutes after the ScheduledBatchJob immediately above? If you use[ the `every` syntax](https://github.com/moove-it/sidekiq-scheduler#schedule-types), you should be able to achieve without a gnarly `cron` statement. Also might be worth looking at [`first_at` or `first_in` syntax ](https://github.com/jmettraux/rufus-scheduler#first_at-first_in-first-first_time) if you want to give yourself some buffer after prod deploy..

Reply:
@batemapf My concern with using the `first_at` or `first_in` as a prod buffer is that if or when the sidekiq process restarts, the delay would occur again.  if everry time sidekiq restarts (no idea how often that happens) we wait 2days to run, then we might inadvertently cause delays in the future, unless we remove the delay once we confirm everything works